packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -15,38 +15,38 @@
 
   display: inline-block;
   position: relative;
-  margin-block: var(--SpaceMarginBlock);
-  margin-inline: var(--SpaceMarginInline);
-  padding-block: var(--SpacePaddingBlock);
-  padding-inline: var(--SpacePaddingInline);
+  margin-block: var(--MarginBlock);
+  margin-inline: var(--MarginInline);
+  padding-block: var(--PaddingBlock);
+  padding-inline: var(--PaddingInline);
   transition-property: color, background-color, border-color, box-shadow;
   transition-duration: var(--TransitionDuration);
   transition-timing-function: var(--TransitionTiming);
   border-width: var(--BorderWidth);
   border-style: var(--BorderStyle);
-  border-color: var(--ColorBorderPrimary);
-  background-color: var(--ColorBackgroundPrimary);
-  color: var(--ColorText);
+  border-color: var(--PrimaryBorderColor);
+  background-color: var(--PrimaryBackgroundColor);
```

Comment:
I think we should take this opportunity to move these out into a new `primaryVariant` class defined below to make this all more clear!

Reply:
I've broken my batch renaming ticket out into separate issues. That way, if we decide we want to standardize on this across components right now, I can describe the work that's happening per-component.